### ISSUE TITLE
String list erases entered data when saved

### ISSUE THREAD

**Issue by josineto** [sveltia/sveltia-cms#168]
Whenever I have a `list` widget with no fields, so a string list, and I type anything in that field **when creating an item** and save, the CMS loses that data and saves it as `[]`, an empty array. I see this happening locally, using Chrome. I didn't test it online.

Below a configuration for one of those fields (the problem happens with 4 different fields with same config, but 2 of them have `required: false`):

```yaml
label:  Categorias
name:   categories
widget: list
i18n:   duplicate
required: true
```

**Comment by kyoshino** [comment]
Will check.

**Comment by kyoshino** [comment]
Fixed in [v0.34.7](https://github.com/sveltia/sveltia-cms/releases/tag/v0.34.7) 🙇🏼

**Comment by josineto** [comment]
Thank you!
